models/collectionswap/ethereum/collectionswap_ethereum_events.sql

```diff
@@ -0,0 +1,163 @@
+{{ config(
+        schema='collectionswap_ethereum',
```

Comment:
more a conversation starter vs. feedback, but is it intentional to have schema in models and leave out of project file? i've said before relatively recently that i think that'd be a good cleanup activity, to do so in models and keep project file clean

Reply:
Yes, it'll also be a requisite in the new NFT setup due to the combined directory structure.

Small steps towards a clean project file :+1: